gpu/test_gpu.py

```diff
@@ -158,8 +158,8 @@ def test_install_gpu_with_mig(self, configuration, machine_suffixes,
         configuration,
         self.INIT_ACTIONS,
         zone=zone,
-        master_machine_type="n1-standard-4",
-        worker_machine_type="n1-standard-4",
+        master_machine_type="g2-standard-4",
```

Comment:
replace all of the GPU_T4 with GPU_L4 if you want to make this change, plz

Reply:
This change is only specific to this test, as GPU_L4 supports mig instances and machine type required is g2 series.

Reply:
makes sense. thanks for clarifying.